config/routes.rb

```diff
@@ -3,6 +3,7 @@
 Moirai::Engine.routes.draw do
   root to: "translation_files#index"
 
+  resources :translations, only: %i[index]
```

Comment:
I like the addition of the translations controller, but I lack a link to it for example from the `/moirai` page. Would this be something to take into consideration or should we just keep it for debugging purposes?

Reply:
At the moment is very ugly. We could for sure link it, but is also not documented at the moment, so let's keep it for debugging purpose for now.